### PR TITLE
ibrahim alduayji

### PR DESCRIPTION
**Pull request by alduayji**

### CODE REVIEW

**Comment by marcwright** [comment]
Marc is grading

**Comment by marcwright** [comment]
Very nice job! 2 out of 2!

The app works with no bugs from what I can see. The code is readable.

- Nice use of a ternary here:

```
        let Rimg = child.data.preview
        ? child.data.preview.images[0].source.url
        : 'https://cdn.comparitech.com/wp-content/uploads/2017/08/reddit-1.jpg';
```


- Think about creating variables for the HTML DOM elements that you use more that once. For example - `document.querySelector("#popTitle")`.

- This could be indented better:

```     
 let newArticle = `
      <article id=${id} class="article">
      <section class="featuredImage">
      <img src="${img}" alt="images/article_placeholder_1.jpg" />
      </section>
      <section class="articleContent">
      <h3>${title}</h3>
      <h6>${author}</h6>
      </section>
      <section class="impressions">
      ${id}
      </section>
      <div class="clearfix"></div>
      </article>
      `
```

File: feedr-starter-code/js/app.js [hunk 1]
```diff
@@ -1,3 +1,104 @@
...
+      let url = article.url;
+      
+
+      // console.log(id);
+      // console.log(catagory);
+      // console.log(description);
+      // console.log(url);
+
+
+      let newArticle = `
```
Comment (marcwright) [inline feedr-starter-code/js/app.js:73]: This could be better indented.

File: feedr-starter-code/js/app.js [hunk 2]
```diff
@@ -1,3 +1,104 @@
...
+      1- find an image
+      2- artical
+      3- catagory
+      4- description 
+      5- the for the url (the actual artical)
+
+*/
+
+var newsAPI = '31b842d19ab647dc94fd31301e4d9475';
+var newsUrl = 'https://newsapi.org/v2/sources?apiKey=31b842d19ab647dc94fd31301e4d9475';
```
Comment (marcwright) [inline feedr-starter-code/js/app.js]: Remember that it's not best practice to push your API keys to GitHub.

Comment (alduayji) [reply feedr-starter-code/js/app.js]: Indeed!

**Pull request closed without merging**